languages/en.php

```diff
@@ -154,6 +154,7 @@
 	'access:write' => "Write access",
 	'access:admin_only' => "Administrators only",
 	'access:missing_name' => "Missing access level name",
+	'access:comments:change' => "There are comments on this content, by changing the access level you will also be changing the access of those comments.  Do you want to continue?",
```

Comment:
@teamwebgalli - you must have missed our 90+ count discussion thread on it :)  I agree it's the expected behavior, but this is my compromise for getting the comment access fix in.

@juho-jaakkola - I agree visibility is better, I used access level for consistency with the label.  I'm certainly open to a better message though.